portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/clay/ClayAlertToast.testcase

```diff
@@ -55,7 +55,10 @@ definition {
 		}
 
 		task ("Then the alert does not disappear when hovered over") {
-            Pause(locator1 = "20100");
+
+			// For LPS-170578, need this pause to wait longer than the auto close time of the fail message
+
+			Pause(locator1 = "20100");
```

Comment:
Hey @mateussandes23 , what happened here? I thought you had said you updated the PR with adding a comment like we discussed in our slack?

I went ahead and made the edit already. But would like to hear what happened on your side.